browser/widevine/BUILD.gn

```diff
@@ -50,43 +50,53 @@ source_set("constants") {
 source_set("unittest") {
   testonly = true
 
-  sources = [ "widevine_cdm_component_installer_unittest.cc" ]
-  deps = [
-    "//testing/gtest",
-    "//third_party/widevine/cdm:buildflags",
-  ]
+  if (enable_widevine_cdm_component) {
+    sources = [ "widevine_cdm_component_installer_unittest.cc" ]
+    deps = [
+      "//testing/gtest",
+      "//third_party/widevine/cdm:buildflags",
+    ]
+  }
 }
 
 source_set("browser_tests") {
   testonly = true
   defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
 
-  sources = [
-    "widevine_permission_request_browsertest.cc",
-    "widevine_prefs_migration_browsertest.cc",
-  ]
+  sources = []
+  deps = []
 
-  deps = [
-    ":constants",
-    ":widevine",
-    "//base",
-    "//brave/browser",
-    "//brave/common",
-    "//brave/components/constants",
-    "//chrome/browser",
-    "//chrome/browser:browser_process",
-    "//chrome/browser/profiles:profile",
-    "//chrome/browser/ui",
-    "//chrome/common",
-    "//chrome/test:test_support_ui",
-    "//components/prefs",
-    "//content/public/browser",
-    "//content/test:test_support",
-    "//testing/gtest",
-    "//third_party/widevine/cdm:buildflags",
-    "//ui/views",
-    "//url",
-  ]
+  if (!is_android) {
+    sources += [ "widevine_prefs_migration_browsertest.cc" ]
```

Comment:
is there an android version of the permission test? Or modify this test to also work on android?

Reply:
We don't need this migration test on Android.
4y ago per-profile widevine related prefs was migrated from profile to local state and android didn't use that prefs at that time. Also I think we can delete migration related code/tests now as 4y passed.

Reply:
I think he's actually talking about `widevine_permission_request_browsertest.cc`, just wrong line.

Reply:
@bridiver The non-desktop specific part of `widevine_permission_request_browsertest.cc` is added back now.